substrate/frame/broker/src/benchmarking.rs

```diff
@@ -1200,6 +1200,33 @@ mod benches {
 		Ok(())
 	}
 
+	#[benchmark]
+	fn remove_assignment() -> Result<(), BenchmarkError> {
```

Comment:
You need to also add this to the weights files. Check the output of clippy (or the clippy CI job)